pytorch_lightning/tuner/batch_size_scaling.py

```diff
@@ -121,7 +121,7 @@ def __scale_batch_reset_params(trainer: 'pl.Trainer', model: 'pl.LightningModule
     trainer.logger = DummyLogger()
     trainer.callbacks = []  # not needed before full run
     trainer.limit_train_batches = 1.0
-    trainer.optimizers, trainer.schedulers = [], []  # required for saving
+    trainer.optimizers, trainer.lr_schedulers = [], []  # required for saving
```

Comment:
my grep powers found that these are also wrong - but unrelated to the DeepSpeed fixes in this PR